### DISCUSSION TITLE
Orders Status from API

### DISCUSSION THREAD

**Discussion by gabrielleal7** [woocommerce/woocommerce#35420] [WooCommerce HPOS] [answered]
Hi, i would like to know if there is a way to get all order statuses via API, do you have an endpoint for that? For example /orders_statuses?

**Comment by barryhughes** [comment, marked as answer]
Hi @gabrielleal7! 

You've landed in the discussion area for the [WooCommerce HPOS (High-Performance Order Storage) project](https://github.com/woocommerce/woocommerce/discussions/categories/woocommerce-hpos) ... it doesn't sound like your question relates directly to that project and so it would be better to [ask this elsewhere](https://woocommerce.com/document/how-to-get-help/) or start by taking a look at [this REST API documentation](https://woocommerce.github.io/woocommerce-rest-api-docs/#introduction).

Of course, if we have misunderstood and this does relate to HPOS, please provide some additional context.

Thanks!
